pages/common/mocha.md

```diff
@@ -0,0 +1,23 @@
+# mocha
+
+> Execute Mocha JavaScript test runner
+
+- Run tests with default configuration or as configured in `mocha.opts`
+
+`mocha`
+
+- Run tests contained at specific paths (files or directories)
+
+`mocha {{path-1}} {{path-2}} {{path-N}}`
```

Comment:
this way we start looking like `man`, you described it too formal.

In 95% of cases it is folder ./test or ./spec. So maybe it is good to mention that.
```
`mocha {{path/to/tests}}`
```